policy/modules/system/systemd.if

```diff
@@ -2843,6 +2843,29 @@ interface(`systemd_map_resolved_exec_files',`
 	allow $1 systemd_resolved_exec_t:file map;
 ')
 
+########################################
+## <summary>
+##	Send and receive messages from
+##	systemd networkd over dbus.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`systemd_dbus_chat_networkd',`
+	gen_require(`
+		type systemd_networkd_t;
+		class dbus send_msg;
+	')
+
+	allow $1 systemd_networkd_t:dbus send_msg;
+	allow systemd_networkd_t $1:dbus send_msg;
+	allow $1 systemd_networkd_t:unix_stream_socket connectto;
+	ps_process_pattern(systemd_networkd_t, $1)
```

Comment:
This looks like 2 or 3 different permission sets. Can you elaborate for what is that needed?